### PR TITLE
Remove unsupported v1alpha5 releases

### PR DESCRIPTION
**Pull request by mboukhalfa**
v1alpha5 is no longer supported this PR remove the mentions of the related release from docs

### CODE REVIEW

**Comment by tuminoid** [comment]
https://github.com/metal3-io/metal3-docs/blob/b68300bba1e992583f447c24ab9594bd7d5fd1d5/docs/user-guide/src/capm3/introduction.md?plain=1#L21 has mention on alpha5 and even older, should those be cleaned away as well?

**Comment by kashifest** [comment]
Latest releases and tags needs to be updated here as well? Can you please do that in the same patch? For example, CAPM3 v1.1.4,  BMO 0.2.0.

File: docs/user-guide/src/version_support.md [hunk 1]
```diff
@@ -31,15 +29,11 @@ Following the same trend, the following tags for BMO and Ironic are available an
 - capm3-v1.1.2 (BMO and Ironic)
 - capm3-v1.1.1
 - capm3-v1.1.0
```
Comment (kashifest) [inline docs/user-guide/src/version_support.md:31]: Some newer tags are missing here also, check here https://quay.io/repository/metal3-io/ironic?tab=tags

**Review by kashifest — APPROVED** [review]

File: docs/user-guide/src/version_support.md [hunk 2]
```diff
@@ -3,43 +3,40 @@
...
 - CAPM3
-    - main
-        - v1beta1
-    - release-1.2
-        - v1beta1
-    - release-1.1
-        - v1beta1
+  - main
+    - v1beta1
+  - release-1.2
```
Comment (furkatgofurov7) [inline docs/user-guide/src/version_support.md:8]: [release-1.3](https://github.com/metal3-io/cluster-api-provider-metal3/tree/release-1.3) is missing

File: docs/user-guide/src/version_support.md [hunk 3]
```diff
@@ -3,43 +3,40 @@
...
 
 Currently, in Metal³ organization only CAPM3 and IPAM follow CAPI release cycles. The supported versions (excluding release candidates) for CAPM3 and IPAM releases are as follows:
 
 - CAPM3
-    - v1beta1
-        - v1.2.0, v1.1.3, v1.1.2, v1.1.1, v1.1.0
-    - v1alpha5
-        - v0.5.5, v0.5.4, v0.5.3, v0.5.2, v0.5.1, v0.5.0
+  - v1beta1
+    - v1.1.4, v1.3.0, v1.2.1, v1.2.0, v1.1.3, v1.1.2, v1.1.1, v1.1.0
```
Comment (furkatgofurov7) [inline docs/user-guide/src/version_support.md:17]:
```suggestion
    -  v1.3.0, v1.2.1, v1.2.0, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0
```

File: docs/user-guide/src/version_support.md [hunk 4]
```diff
@@ -3,43 +3,40 @@
...
 - IPAM
-    - v1alpha1
-        - v1.2.0, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0, v0.1.2, v0.1.1, v0.1.0
+  - v1alpha1
+    - v1.1.5, v1.3.0, v1.2.1, v1.2.0, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0, v0.1.2, v0.1.1, v0.1.0
```
Comment (furkatgofurov7) [inline docs/user-guide/src/version_support.md:20]:
```suggestion
    - v1.3.0, v1.2.1, v1.2.0, v1.1.5, v1.1.4, v1.1.3, v1.1.2, v1.1.1, v1.1.0, v0.1.2, v0.1.1, v0.1.0
```

File: docs/user-guide/src/version_support.md [hunk 5]
```diff
@@ -3,43 +3,40 @@
...
 
 The compatability of IPAM and CAPM3 API versions with CAPI is discussed [here](https://github.com/metal3-io/ip-address-manager#compatibility-with-cluster-api).
 
 Since BMO and Ironic do not follow similar release cycles they are backward compatible. However, we used to tag the BMO and Ironic code base whenever we do a release in CAPM3 prior to CAPM3 release `v1.1.3`. The tags used to have a prefix **capm3-** and the suffix was always the corresponding capm3- release version. So for example, if we cut a `v1.0.0` release for CAPM3 we created a tag in the BMO and Ironic code base with `capm3-v1.0.0`. Please note, currently that is applicable only for Ironic starting from CAPM3 release `v1.1.3` and onwards.
 Following the same trend, the following tags for BMO and Ironic are available and supported (image tags):
 
+- capm3-v1.1.4 (Ironic Only)
```
Comment (furkatgofurov7) [inline docs/user-guide/src/version_support.md:27]: Move it under `- capm3-v1.2.0 (Ironic Only)`

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: docs/user-guide/src/version_support.md (hunk 1 above)
Comment (tuminoid) [reply docs/user-guide/src/version_support.md:34]: Fixed in #330

**Comment by tuminoid** [comment]
Superseded by #330. Everything is included in that.

/close

**Pull request closed without merging**